helpers/package.json

```diff
@@ -13,15 +13,16 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://gitlab.com/tzstamp/helpers.git"
+    "url": "https://github.com/marigold-dev/tzstamp.git",
+    "directory": "helpers"
```

Comment:
nice, didn't knew about this propertty

Reply:
Thanks! I got it here: https://docs.npmjs.com/cli/v7/configuring-npm/package-json#directories